docs/source/scores/guidelines.rst

```diff
@@ -5,8 +5,62 @@ DCML Score Conventions
 .. contents:: Contents
    :local:
 
+General Principles
+==================
+
+These guidelines are intended to ensure that the scores produced by and for the DCML are consistent and high-quality
+encodings of some print or manuscript edition. Although they represent digital editions, first and foremost they
+are to be considered as *research data*. As such, we neglect details of layout in favour of rigorous conventions
+regarding the semantically correct encoding of score elements. Semantic, in this context, means that wherever
+possible the various score elements are to be created with the corresponding tools and features that the
+typesetting software MuseScore provides. For example, a ``cresc.`` performance indication must not be encoded
+as a "normal" staff text: It has the semantics of a dynamic marking and is available as such in the "Lines" palette.
+A large part of these guidelines is concerned with stipulating how various score elements are to be encoded.
+
+Get the latest version of MuseScore 3
+-------------------------------------
+
+Unfortunately, MuseScore 4 does not allow opening an ``.mscx`` (i.e. uncompressed) MuseScore file and store it back.
+As long as `this issue <https://github.com/musescore/MuseScore/issues/15995>`__ is not resolved, we are sticking to
+the latest version of MuseScore 3, which is 3.6.2. You can download it from the
+`official GitHub page <https://github.com/musescore/MuseScore/releases/tag/v3.6.2>`__.
+Head to the link above and download the installer for your operation system. If
+you're using Linux you can downgrade the package in your package manager or, instead, download and execute the AppImage.
+
+
+Organization into movements or pieces
+-------------------------------------
+
+A major use case for the scores is the study of tonal music in all its aspects. It has proven worthwhile to
+organize the scores into units which, tonally, stand for themselves in one way or another. Three aspects play a
+role when deciding into how many different files to split a musical expression:
+
+#. **measure numbering**: it should make sense to start with 0 or 1
+#. **global key**: it should make (at least some) sense to analyze the contents under a single global tonic
```

Comment:
Other things equal, if it makes sense to analyze (part of) a work under a single global tonic, that speaks in favor of storing it in a single file.